frame/assets/src/lib.rs

```diff
@@ -24,41 +24,72 @@
 //! The Assets module provides functionality for asset management of fungible asset classes
 //! with a fixed supply, including:
 //!
-//! * Asset Issuance
-//! * Asset Transfer
-//! * Asset Destruction
+//! * Asset Issuance (Minting)
+//! * Asset Transferal
+//! * Asset Freezing
+//! * Asset Destruction (Burning)
 //!
 //! To use it in your runtime, you need to implement the assets [`Trait`](./trait.Trait.html).
 //!
 //! The supported dispatchable functions are documented in the [`Call`](./enum.Call.html) enum.
 //!
 //! ### Terminology
 //!
-//! * **Asset issuance:** The creation of a new asset, whose total supply will belong to the
-//!   account that issues the asset.
-//! * **Asset transfer:** The action of transferring assets from one account to another.
-//! * **Asset destruction:** The process of an account removing its entire holding of an asset.
-//! * **Fungible asset:** An asset whose units are interchangeable.
-//! * **Non-fungible asset:** An asset for which each unit has unique characteristics.
+//! * **Admin**: An account ID uniquely privileged to be able to unfreeze (thaw) an account and it's
+//!   assets, as well as forcibly transfer a particular class of assets between arbitrary accounts
+//!   and reduce the balance of a particular class of assets of arbitrary accounts.
+//! * **Asset issuance/minting**: The creation of a new asset, whose total supply will belong to the
+//!   account that issues the asset. This is a privileged operation.
+//! * **Asset transfer**: The reduction of the balance of an asset of one account with the
+//!   corresponding increase in the balance of another.
+//! * **Asset destruction**: The process of reduce the balance of an asset of one account. This is
+//!   a privileged operation.
+//! * **Fungible asset**: An asset whose units are interchangeable.
+//! * **Issuer**: An account ID uniquely privileged to be able to mint a particular class of assets.
+//! * **Freezer**: An account ID uniquely privileged to be able to freeze an account from
```

Comment:
Just a comment: The asymmetry of the freezer being able to freeze but not thaw accounts is strange to me, and not standard based on the most common asset contracts that I am familiar with.

Reply:
Freezing is a strictly less sensitive operation than thawing - it could be reasonable for a reserve-backed coin operator to many people employed with the permissions to be able to issue a temporary freeze based on a fraud/LEA claim, yet require unfreezing to go via a compliance department.